src/components/DataTable/DataTable-story.js

```diff
@@ -98,81 +15,7 @@ storiesOf('DataTable', module)
       You can find more detailed information surrounding usage of this component
       at the following url: ${readmeURL}
     `,
-    /* eslint-enable no-useless-escape */
-    () => (
-      <DataTable
-        rows={initialRows}
-        headers={headers}
-        render={({
-          rows,
-          headers,
-          getHeaderProps,
-          getSelectionProps,
-          getBatchActionProps,
-          onInputChange,
-          selectedRows,
-        }) => (
-          <TableContainer title="DataTable">
-            <TableToolbar>
-              <TableBatchActions {...getBatchActionProps()}>
-                <TableBatchAction onClick={batchActionClick(selectedRows)}>
-                  Ghost
-                </TableBatchAction>
-                <TableBatchAction onClick={batchActionClick(selectedRows)}>
-                  Ghost
-                </TableBatchAction>
-                <TableBatchAction onClick={batchActionClick(selectedRows)}>
-                  Ghost
-                </TableBatchAction>
-              </TableBatchActions>
-              <TableToolbarSearch onChange={onInputChange} />
-              <TableToolbarContent>
-                <TableToolbarAction
-                  iconName="download"
-                  iconDescription="Download"
-                  onClick={action('TableToolbarAction - Download')}
-                />
-                <TableToolbarAction
-                  iconName="edit"
-                  iconDescription="Edit"
-                  onClick={action('TableToolbarAction - Edit')}
-                />
-                <TableToolbarAction
-                  iconName="settings"
-                  iconDescription="Settings"
-                  onClick={action('TableToolbarAction - Settings')}
-                />
-                <Button onClick={action('Add new row')} small kind="primary">
-                  Add new
-                </Button>
-              </TableToolbarContent>
-            </TableToolbar>
-            <Table>
-              <TableHead>
-                <TableRow>
-                  <TableSelectAll {...getSelectionProps()} />
-                  {headers.map(header => (
-                    <TableHeader {...getHeaderProps({ header })}>
-                      {header.header}
-                    </TableHeader>
-                  ))}
-                </TableRow>
-              </TableHead>
-              <TableBody>
-                {rows.map(row => (
-                  <TableRow key={row.id}>
-                    <TableSelectRow {...getSelectionProps({ row })} />
-                    {row.cells.map(cell => (
-                      <TableCell key={cell.id}>{cell.value}</TableCell>
-                    ))}
-                  </TableRow>
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
-        )}
-      />
-    )
+    require('./stories/default').default
```

Comment:
Should use ESM import.

Reply:
Any reason?

Reply:
Given the codebase is standardized on ESM, avoiding CJS `require()` helps our codebase look more coherent. Doing so avoids the kind of bugs e.g. one like a recent date picker PR introduced.

Reply:
Have a link to the bug for date picker? Would love to understand that, didn't realize something had happened 😮

Reply:
Here it is: https://github.com/carbon-design-system/carbon-components-react/pull/869#issuecomment-387591240